### PR TITLE
Fix typo esimtate -> estimate

### PR DESCRIPTION
**Pull request by langestefan**

### CODE REVIEW

File: src/show.jl [hunk 1]
```diff
@@ -121,10 +121,10 @@ function show(io::IO, model::PEtabModel)
     print(io, styled"$(header)$(opt)")
 end
 function show(io::IO, prob::PEtabODEProblem)
-    @unpack probinfo, model_info, nparameters_esimtate = prob
+    @unpack probinfo, model_info, nparameters_estimate = prob
     name = model_info.model.name
     nstates = @sprintf("%d", length(unknowns(model_info.model.sys_mutated)))
```
Comment (sebapersson) [inline src/show.jl:126]: With the change here, does PEtabODEProblem still print correctly after being built?

**Review by sebapersson — CHANGES_REQUESTED** [review]

File: src/show.jl (hunk 1 above)
Comment (langestefan) [reply src/show.jl:126]: no, the same typo is made in several other files:

https://github.com/search?q=repo%3Asebapersson%2FPEtab.jl%20esimtate&type=code

Comment (sebapersson) [reply src/show.jl:126]: Could you do a search to change it in all files (quite straightforward in vscode), the most critical place to fix this is in the `PEtabODEProblem` struct

**Comment by sebapersson** [comment]
Fixed by #237

**Pull request closed without merging**